app/components/document-delete-button.js

```diff
@@ -0,0 +1 @@
+export { default } from "ember-alexandria/components/document-delete-button";
```

Comment:
You can delete this file as this is not really a component worth exporting to the host app, since this is an engine there are normally no component exports to the host app.

Reply:
Done